src/creators/createTypes/index.js

```diff
@@ -5,8 +5,8 @@
  * stringArrayToObject(['A', 'B', 'C']) // { A: 'A', B: 'B', C: 'C' }
  */
 function stringArrayToObject(actionsArray, namespace) {
-  if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
-    throw new Error('Action names must be an array of strings.');
+  if (!Array.isArray(actionsArray) || actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
```

Comment:
Mmm, does this works after transpiling ?
For reference to my doubt:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill

Reply:
It seems like this has been supported for a while now
http://kangax.github.io/compat-table/es5/#test-Array.isArray

(Go to Array methods -> Array.isArray)